docs/03_tutorials/21_tic-tac-toe-game-smart-contract-Testnet.md

```diff
@@ -296,7 +296,7 @@ public:
     name getWinner(const game &currentGame);
 ```
 
-The complete tictactoe.hpp file can be downloaded from github here: [Tic-tac-toe tutorial hpp source.](https://github.com/EOSIO/welcome/tree/master/docs/03_tutorials/src/tictactoe/tictactoe.hpp "tic-tac-toe example hpp")  
+The complete tictactoe.hpp file can be downloaded from github here: [Tic-tac-toe tutorial hpp source.](https://github.com/EOSIO/welcome/blob/release/2.0.x/src/tictactoe/tictactoe.hpp  "tic-tac-toe example hpp") 
```

Comment:
Update or keep `release/2.0.x` if needed depending on which option (a,b,c) we select.